### PR TITLE
[ci skip] Fix link rubyonrails.org/core => http://rubyonrails.org/community

### PR DESCRIPTION
**Pull request by mechanicles**
Fix link rubyonrails.org/core => http://rubyonrails.org/community/#core

### CODE REVIEW

File: guides/source/contributing_to_ruby_on_rails.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ learn about Ruby on Rails, and the API, which serves as a reference.
 You can help improve the Rails guides by making them more coherent, consistent or readable, adding missing information, correcting factual errors, fixing typos, or bringing them up to date with the latest edge Rails.
 
 You can either open a pull request to [Rails](https://github.com/rails/rails) or
-ask the [Rails core team](http://rubyonrails.org/core) for commit access on
+ask the [Rails core team](http://rubyonrails.org/community) for commit access on
```
Comment (maclover7) [inline guides/source/contributing_to_ruby_on_rails.md]: http://rubyonrails.org/community/#core is a better link, autoscrolls to the core section on the page :)

Comment (mechanicles) [inline guides/source/contributing_to_ruby_on_rails.md]: Agreed. Going to update it.

**Comment by mechanicles** [comment]
@maclover7 I have updated it as per your suggestion. Please check.

**Comment by maclover7** [comment]
awesome, thanks @mechanicles!

**Comment by maclover7** [comment]
:+1: Thank you for your contribution!

**Comment by maclover7** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**